src/common/nullpo.h

```diff
@@ -19,52 +19,48 @@
  *----------------------------------------------------------------------------
  */
 /*======================================
- * Null�`�F�b�N �y�� ���o�͌� return
- *�E�W�J�����if�Ƃ�return�����o��̂�
- *  ��s�P�̂Ŏg��Ă��������B
- *�Enullpo_ret(x = func());
- *  �̂悤�Ȏg�p�@��z�肵�Ă��܂��B
+ * Check for NULL pointer and output information
```

Comment:
Should have "after return" at the end as you can see in the japanese text
The additional comment that was lost here is a recommendation to use a single line of code for this kind of checks and a note that this also works for/with functions.